ftw/linkchecker/browser/dashboard.py

```diff
@@ -20,6 +20,18 @@ def __init__(self, context, request):
         self.request = request
         self.context = context
 
+        self._persisted_data = self._get_annotation()
+        self._timestamp = self._persisted_data.get('timestamp', 0)
+        self._reports = self._load_and_sort_fileblock_reports()
+
+        self._latest_report_df = self._get_latest_report_df_from_excel()
+        if isinstance(self._latest_report_df, pd.DataFrame):
+            # there is new data -> update/merge data
+            pass
+        else:
+            # no data or no new data found
+            pass
```

Comment:
This block does nothing, so I would remove it.

Reply:
The PR was not yet ready to review. I'm still implementing that ;-)

Reply:
Oooops, my bad 🙈